libnmstate/nm/plugin.py

```diff
@@ -159,3 +180,19 @@ def _remove_ovs_bridge_unsupported_entries(iface_info):
     iface_info.pop(Interface.MTU, None)
 
     return iface_info
+
+
+def _nm_utils_decode_version():
+    v = NM.utils_version()
+    return (
+        str((v >> 16) & 0xFF)
+        + "."
+        + str((v >> 8) & 0xFF)
+        + "."
+        + str((v) & 0xFF)
+    )
```

Comment:
Please use `NM.MAJOR_VERSION`, `NM.MINOR_VERSION` and `NM.MICRO_VERSION`.

Reply:
Done, thank you!